lib/src/booleans/boolean_contains.dart

```diff
@@ -0,0 +1,206 @@
+import 'package:turf/turf.dart';
+
+import 'boolean_point_in_polygon.dart';
+import 'boolean_point_on_line.dart';
+
+/// Boolean-contains returns True if the second geometry is completely contained
+/// by the first geometry.
+/// The interiors of both geometries must intersect and, the interior and
+/// boundary of the secondary must not intersect the exterior of the primary.
+/// Boolean-contains returns the exact opposite result of the `turf/boolean-within`.
+/// example:
+/// ```dart
+/// var line = LineString(coordinates: [Position.of([1, 1]), Position.of([1, 2]), Position.of([1, 3]), Position.of([1, 4])]);
+/// var point = Point(cooridantes:Position.of([1, 2]));
+/// booleanContains(line, point);
+/// //=true
+/// ```
+bool booleanContains(GeoJSONObject feature1, GeoJSONObject feature2) {
+  var geom1 = feature1 is Feature ? feature1.geometry : feature1;
+  var geom2 = feature2 is Feature ? feature2.geometry : feature2;
```

Comment:
getGeom can also produce a List<GeometryTypes> in case of GeometryCollection. That would make catching the Exception difficult. Wouldn't it?

Reply:
Then getGeom is not good. Let's refactor getGeom to return `GeometryObject`, so that we don't return the `List<GeometryType>` of the `GeometryCollection`, but the `GeometryCollection` itself

Reply:
done